.github/workflows/cr.yaml

```diff
@@ -0,0 +1 @@
+make-release-latest: false
```

Comment:
This needed to be added as a config because the github action doesn't support this parameter yet

Reply:
You should be able to do this be adding `CR_MAKE_RELEASE_LATEST: false` to the `env` block in the _Run chart-releaser_ step, which will follow the existing pattern.

Reply:
makes sense, thank you for the suggestion :+1: